# 
geometry_msgs/Point[] start_points
geometry_msgs/Point[] finish_points
float32 dist_to_finish
float32 dist_to_start
